indico/modules/events/models/principals.py

```diff
@@ -24,6 +24,7 @@ class EventPrincipal(PrincipalPermissionsMixin, db.Model):
     allow_networks = True
     allow_event_roles = True
     allow_category_roles = True
+    allow_registration_form = True
```

Comment:
I think it makes sense for session/contribution/attachment ACLs to allow regforms as well (anything where granting read access makes sense)

Reply:
okay, I will add it :+1: